variations.py

```diff
@@ -1,14 +1,49 @@
 from copy import deepcopy
 
 from .booking import Unit
+from .booking import dataset_from_crownoutput
 from .booking import dataset_from_artusoutput
 from .utils import Selection
 from .utils import Variation
 
 import logging
 logger = logging.getLogger(__name__)
 
+class ChangeDatasetCrown(Variation):
```

Comment:
Maybe you could name it more appropriately as the change of the dataset is the only thing it does not do any more :D
```suggestion
class ReplaceVariable(Variation):
```